test/test_installation.py

```diff
@@ -181,9 +199,16 @@ def install_umf(self) -> None:
                     f"Error: Installation directory '{self.install_dir}' is not empty"
                 )
 
-        install_cmd = f"cmake --install {self.build_dir} --config {self.build_type.title()} --prefix {self.install_dir}"
+        major, minor = self._get_cmake_major_minor_version()
+        if major <= 3 and minor < 15:
+            install_cmd = f"cmake --build {self.build_dir} --config {self.build_type.title()} --prefix {self.install_dir} --target install"
```

Comment:
This command can be used for all CMake versions. The `else` clause is unnecessary, I think.

Reply:
done